client/models.py

```diff
@@ -63,25 +68,41 @@ class Volunteer(Timestampable):
         ("RAD_TV", "רדיו וטלוויזיה"),
         ("OTHR", "אחר")
     )
+    TYPES = (
+        ("NIGHBORHOOD_COORDINATOR", "רכז שכונה"),
+        ("CITY_COORDINATOR", "רכז עיר"),
+        ("STAFF", "מטה"),
+        ("HAMAL", "חמל"),
+        ("PROJECT", "פרויקט"),
+        ("CHILD_CARE", "משפחתון"),
+        ("MISSIONS", "משימות")
+    )
 
-    tz_number = models.CharField(max_length=11, blank=True)
-    full_name = models.CharField(max_length=200)
+    tz_number = models.CharField(max_length=ID_LENGTH, blank=True)
+    first_name = models.CharField(max_length=DEFAULT_MAX_FIELD_LENGTH, default="abcdef")
+    last_name = models.CharField(max_length=DEFAULT_MAX_FIELD_LENGTH, default="abcdef")
```

Comment:
`
```suggestion
    first_name = models.CharField(max_length=DEFAULT_MAX_FIELD_LENGTH, blank=True)
    last_name = models.CharField(max_length=DEFAULT_MAX_FIELD_LENGTH, blank=True)
```
consider blank=True without default